table/test/special_chars.ts

```diff
@@ -12,7 +12,7 @@ Deno.test("table - special chars - chinese characters", () => {
       .toString(),
     `
 名字为 豆贝 尔维了 cell2 cell3
-cell1              cell2 豆贝 
+cell1              cell2 豆贝${" "}
```

Comment:
Yes, that is a bit annoying. also, if border is not enabled and the terminal is resized, the spaces are moved to the next row, which destroys the layout of the table.
I think it would be better anyway not to fill the row with spaces if the border is not enabled.
I think we should try to fix this in a separate pr.